src/coreclr/CMakeLists.txt

```diff
@@ -203,7 +203,9 @@ include_directories("${GENERATED_INCLUDE_DIR}")
 include_directories("hosts/inc")
 include_directories("minipal")
 
+if(FEATURE_INTERPRETER)
```

Comment:
This doesn't work due to the FEATURE_INTERPRETER containing generator expressions. See my other comment for a way to exclude the clrinterpreter

Reply:
Will remove, thanks.

Reply:
Maybe I need to make the install_clr command in the cmakelists conditional?

Reply:
I get the error on Linux too. I was testing it on Windows before. Unfortunately, there doesn't seem to be a way to prevent the install when the FEATURE_INTERPRETER does not evaluate to 1. The. So it seems the only way we have is ifdefing out the whole body of the compiler.cpp to prevent build breaks and live with the fact that we get a crippled clrinterpreter instead of none.

Reply:
I'll revert the two cmake changes for now then, and keep the ifdef in compiler.cpp.